src/bin/tikv-ctl.rs

```diff
@@ -174,9 +174,27 @@ trait DebugExecutor {
         println!("entry {:?}", entry);
         println!("msg len: {}", data.len());
 
-        let mut msg = RaftCmdRequest::new();
-        msg.merge_from_bytes(&data).unwrap();
-        println!("{:?}", msg);
+        if data.is_empty() {
+            println!("this is a empty entry, it should be sent when a peer become leader");
```

Comment:
I think we can remove this. above output have already told the message is empty.